staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -358,6 +388,17 @@ func (t *tracker) Watch(gvr schema.GroupVersionResource, ns string, opts ...meta
 	if err != nil {
 		return nil, err
 	}
+	// By default, emulate the traditional behavior of the tracker and don't deliver
+	// *any* existing objects. This ensures compatibility which users
+	// which don't pass options and then also don't expect the additional objects.
+	resourceVersion := int64(-1)
+	if len(opts) > 0 && opts[0].ResourceVersion != "" {
```

Comment:
Can we use the resource version helpers instead here? We can use https://pkg.go.dev/k8s.io/apimachinery/pkg/util/resourceversion to compare. Compare the resource version against itself to validate whether or not the resource version is valid. We shouldn't bound this to an int64.

Reply:
Why is that better? The ResourceVersion here is generated by this code, it can pick int64 as the internal representation and then base its logic on that.

My understanding is that the helpers are for clients of an API server. This here is the server.